auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -273,6 +313,16 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
     return LaunchDarklyService.getFlag(LDFlags.EnableMandatoryAddress) || false
   }
 
+  private get enableBusinessTable (): boolean {
+    // disabling table manually for early stages of development.
+    return false // LaunchDarklyService.getFlag(LDFlags.EnableBusinessTable) || false
```

Comment:
Will you leave it hardcoded for now?

Reply:
Yea, until the questions surrounding the accumulation of the NR data is in place i felt we wouldn't even want to enable the FF in Dev. 
I just did this so the code is in place and easy to implement once we have the model updates and a solution to getting the NR data. 
At that point we will remove this and toggle dev environment for dev/testing etc.

Reply:
I'm OK with this.